book.rb

```diff
@@ -0,0 +1,27 @@
+# class Book
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book, person, date)
+    Rental.new(book, person, date)
+  end
+end
+
+# class Rental
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(book, person, date)
+    @date = date
+    @person = person
+    @book = book
+    book.rentals.push(self)
+    person.rentals.push(self)
+  end
+end
```

Comment:
- [ ] It appears that the Render class is a separate entity from the Book class and should therefore be placed in a different file  📁. To ensure this, please create a new file 📂✏️ for the Render class and place it there. ✏️🖊️